packages/devtools_app/lib/src/flutter/scaffold.dart

```diff
@@ -81,10 +81,7 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
     with TickerProviderStateMixin {
   /// A tag used for [Hero] widgets to keep the [AppBar] in the same place
   /// across route transitions.
-  static const String _appBarTag = 'DevTools AppBar';
-
-  AnimationController appBarAnimation;
-  CurvedAnimation appBarCurve;
+  static const Object _appBarTag = 'DevTools AppBar';
```

Comment:
why did this change from String to Object

Reply:
It's used as an `Object` later on, so I just updated the type to reflect the intend.